Kernel/AspectSymfonyKernel.php

```diff
@@ -37,7 +38,7 @@ protected function configureAop(AspectContainer $container)
     public function init(array $options = [])
     {
         // it is a quick way to check if loader was enabled
-        $wasDebugEnabled = class_exists(DebugClassLoader::class, false);
+        $wasDebugEnabled = class_exists('\Symfony\Component\Debug\DebugClassLoader', false);
```

Comment:
No, it isn't required. Technically it's just FQN of class or even namespace that was previously imported. See https://3v4l.org/qptdT for demo. You can use `::class` for free, it's just parser-level constant with string.